### DISCUSSION TITLE
Avoid batch mode warnings in case of non-packet procesing elements

### DISCUSSION THREAD

**Discussion by piotrjurkiewicz** [tbarbette/fastclick#306] [Ideas]
Is there any point in printing batch mode warnings for elements which do not process packets?

    click_driver@@ControlSocket won't be in batch mode because no element produces or sends batches to it.
    click_driver@@ControlSocket@1 won't be in batch mode because no element produces or sends batches to it.

Can FastClick automatically detect such elements and do not print warnings for them?

**Comment by tbarbette** [comment]
No...

**Comment by tbarbette** [comment]
I meant it's not currently done but I'll do it.
